assets/src/edit-story/components/panels/layer/useLayerReordering.js

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
I did some testing and seeing couple of questions:

/1/ I can seem to reorder from the position B+2 (second element from the background layer) to B+1. I can see the separator highlighted right above the background element, but nothing happens.
/2/ I couldn't make multi-element reordering work at all. Is this part of this PR? As soon as I do pointerdown to start dragging, the multi-selection is unselected.

Reply:
Ad. 1: Works for me?

Ad. 2: [Multi-layer reordering is not v1.0](https://xwp.slack.com/archives/CCCAUAH6F/p1576278548024400?thread_ts=1576039887.124500&cid=CCCAUAH6F).

Reply:
Ad. 1: Maybe you have a page with a background image. Haven't actually tried those yet, as I haven't tried the PR with those. Will try it out and see if there's some kinks to be worked out - there most likely is!

Reply:
> Ad. 2: Multi-layer reordering is not v1.0.

Got it. So it's essentially for `delete` only.

> Ad. 1: Maybe you have a page with a background image. 

Yes. I can confirm: it's only broken when the background image is available.

Reply:
> > Ad. 2: Multi-layer reordering is not v1.0.
> 
> Got it. So it's essentially for `delete` only.

And all the other shortcuts that's available for multi-selection (moving, etc). Plus you get the correct inspector design panels for multi-selection context.

> > Ad. 1: Maybe you have a page with a background image.
> 
> Yes. I can confirm: it's only broken when the background image is available.

Will test it and fix.

Reply:
So, we are not anticipating at all needing to go up/down using keyboard errors over this list?

Reply:
> So, we are not anticipating at all needing to go up/down using keyboard arrows over this list?

No, we do not. At least that's what's I talked with Sam about. We might later find out that's wrong, but that's a problem for then 😉.

Reply:
> Yes. I can confirm: it's only broken when the background image is available.

Hm, I can't seem to recreate any problems here. It works for me also with a background image. I can drag all the non-background layers to any position and they reorder correctly every time.

Reply:
> Yes. I can confirm: it's only broken when the background image is available.

I was wrong. Seemingly randomly it seems to sometimes to work and sometimes not. See the demo:
![ezgif-2-305d9b6096e1](https://user-images.githubusercontent.com/726049/74291729-a99e3f00-4cea-11ea-9860-466fde5b01cb.gif)

Reply:
> > So, we are not anticipating at all needing to go up/down using keyboard arrows over this list?
> 
> No, we do not. At least that's what's I talked with Sam about. We might later find out that's wrong, but that's a problem for then 😉.

Hmm. I guess this is for another time. But I thought part of the issue was to allow selection with keyboard for things that might be harder to reach on the canvas because they are "mostly" behind some other things...

Reply:
But if you use keyboard, it doesn't matter if it's visually covered on the page. You'd just tab to it anyway (when we implement that).

Reply:
> I was wrong. Seemingly randomly it seems to sometimes to work and sometimes not. See the demo:

I can't reproduce this in anyway whatsoever. Could it be that you page is somehow corrupted? Or can you recreate it with a new page fairly easily?

Reply:
> I can't reproduce this in anyway whatsoever. Could it be that your page is somehow corrupted? Or can you recreate it with a new page fairly easily?

Ok. Let's proceed with merge. If it's indeed a bug, it will popup again.